Documentation/contributing/development/contributing_guide.rst

```diff
@@ -327,6 +327,11 @@ steps 1 to 2 times per day. Works best if done first thing in the working day.
 #. If this PR was opened by a non-committer (e.g. external contributor) please
    assign yourself to that PR and make sure to keep track the PR gets reviewed
    and merged. This may extend beyond your assigned week for Janitor duty.
+
+   If the contributor is a Cilium committer, then they are responsible for
+   getting the PR in a ready to be merged state by adding the ``ready-to-merge``
+   label so that the janitor can merge it in step #9.
```

Comment:
This reads a little like committers should simply make sure to add the label, so merging is ready. It's probably obvious that reviews/CI must be addressed before that, but I would mention it explicitly to avoid any confusion.

```suggestion
   If the contributor is a Cilium committer, then they are responsible for
   getting the PR in a ready to be merged state by adding the ``ready-to-merge``
   label, once all reviews have been addressed and CI checks are successful,
   so that the janitor can merge it in step #9.
```

Also not sure if `step #9` is future-proof, seeing that we don't manually number list items. Maybe just “see below”?

Reply:
I didn't mention it as it's already in the [pull requests review process for committers](https://docs.cilium.io/en/v1.9/contributing/development/contributing_guide/#pull-requests-review-process-for-committers) section, but I think it's a good call to state it again here :+1: 

> Also not sure if step #9 is future-proof, seeing that we don't manually number list items. Maybe just “see below”?

yep, agree also on that :+1:

Reply:
Yeah this paragraph is not supposed to detail the process for standard PRs, but I just didn't want it to introduce any ambiguity for users who would only read this section. Thanks for the update :+1:.